services/minespace-web/src/components/pages/Project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from "react";
+import { Row, Col, Typography, Button, Alert, Badge } from "antd";
+import Callout from "@/components/common/Callout";
+import { CALLOUT_SEVERITY } from "@mds/common/constants/strings";
+import CoreTable from "@mds/common/components/common/CoreTable";
+import { getProject } from "@mds/common/redux/selectors/projectSelectors";
+import { useSelector } from "react-redux";
+
+import { getPermits } from "@mds/common/redux/selectors/permitSelectors";
+import {
+  renderTextColumn,
+  renderDateColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import { IAuthorizationSummaryColumn } from "@mds/common/interfaces";
+import { Link } from "react-router-dom";
+
+import {
+  getDropdownProjectSummaryPermitTypes,
+  getTransformedProjectSummaryAuthorizationTypes,
+} from "@mds/common/redux/selectors/staticContentSelectors";
+import { ColumnsType } from "antd/es/table";
+import * as routes from "@/constants/routes";
+import { useHistory } from "react-router-dom";
```

Comment:
I'm seeing multiple imports from the same source- there's another one from react-router-dom that can be combined